net-misc/gallery-dl/gallery-dl-1.15.2-r1.ebuild

```
@@ -4,6 +4,7 @@
 EAPI=7
 
 PYTHON_COMPAT=(python3_{7,8})
+PYTHON_REQ_USE=(sqlite ssl xml)
```

This doesn't seem to work, you can test it by emerging any python without these USE flags then emerging gallery-dl.

```suggestion
PYTHON_REQ_USE="sqlite,ssl,xml"
```

---

Oh, ok